packages/shared/src/utils/parseURL.ts

```diff
@@ -1,4 +1,14 @@
 import parser from 'anchorme'
-export function parseURL(string: string) {
-    return parser.list(string).map((x) => x.string)
+export function parseURL(text: string) {
+    return parser
+        .list(text)
+        .map((x) => x.string)
+        .filter((y) => {
+            try {
+                new URL(y)
+                return true
+            } catch {
+                return false
+            }
+        })
```

Comment:
plz extract as function, `.filter(isValidURL)`

Reply:
I think we're ok with this as a temporary fix